### DISCUSSION TITLE
Add option to treat backspace as ctrl-backspace

### DISCUSSION THREAD

**Discussion by precompute** [monkeytypegame/monkeytype#1281] [Ideas] [answered]
This would make the user type the entire word instead of being able to quickly rectify the incorrect letter(s).

**Comment by Miodec** [comment, marked as answer]
I dont understand the point of this

**Reply by unrealapex** [reply]
That isn't practical for large words.

**Reply by precompute** [reply]
It's an added level of difficulty that ensures one practises the entire word insead of just the incorrect letter(s).

**Reply by unrealapex** [reply]
Thanks for the clarification. I do think that would be useful, but I think you can already do that. 



https://user-images.githubusercontent.com/79121360/115577994-b963cc80-a279-11eb-9162-2b28dd5e4f77.mp4


Check with @Miodec.

**Reply by precompute** [reply]
I don't understand what you're trying to demonstrate with the video.  Does the option already exist?

**Reply by unrealapex** [reply]
Are you referring to a  mode where instead of being able to `backspace` and `ctrl + backspace` you can only `ctrl + backspace` ?

**Reply by frogfrogfrogfrogfrog** [reply]
@t-e-r-m I'm pretty sure that @UnrealApex is showing that you could simply do this manually without the need of a feature being implemented in Monkeytype. I personally think that developing the habit of doing ctrl + backspace is better than using the idea you've provided.

**Reply by unrealapex** [reply]
Yes indeed.

**Reply by precompute** [reply]
Okay, thanks.  

> Are you referring to a mode where instead of being able to backspace and ctrl + backspace you can only ctrl + backspace 

pressing backspace would always generate ctrl+backspace.

**Reply by unrealapex** [reply]
I think it wouldn't be useful as a mode, but it would be a good funbox.
